### ISSUE TITLE
Define index file schema to have GWAS data searchable from Covid portal

### ISSUE THREAD

**Issue by twhetzel** [EBISPOT/goci#119]
Define schema and create index file for GWAS entry to add to the EBI Search index based on documentation here [1]. This is needed in order to have GWAS data included in the Covid portal [2].

[1] https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EXT&title=EBI+Search+Engine
[2] https://www.covid19dataportal.org/

**Comment by twhetzel** [comment]
Collect information on what is needed to have GWAS data indexed in the Covid portal [1]. Is there a data format to use for the information, what GWAS Catalog entity will be indexed, e.g. Publication, Study, Variant, etc

[1] https://www.covid19dataportal.org/

**Comment by twhetzel** [comment]
For reference, this is the spec for the GWAS Catalog search index: 
https://docs.google.com/document/d/1i7eDTVJwvdCOcL5Rptbg4B-vYJ2LX35AZyfaRZRsLb8/edit

**Comment by twhetzel** [comment]
Following discussion with LJWH, decided the following:
- Name field: GWAS: {Reported Trait} {AccessionId}
- Description: Study of {Discovery(initial) sample description} from {First Author Last Name} PMID: {PMID} 

Where Reported Trait, AccessionId, Discovery sample description, Author Name, and PMID are values from the database. The title text will be linked to the GWAS Catalog Study page, e.g. https://www.ebi.ac.uk/gwas/studies/GCST002709

Example: 
```
GWAS: Electroencephalogram traits GCST002709
Study of 2,383 European ancestry adolescents and 1,643 European ancestry adults from 1,613 families published by Malone SM, PMID: 25387704 
```
